src/PieChart/PieChart.webmodeler.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, createElement } from "react";
+
+import { PieChart } from "./components/PieChart";
+import PieChartContainer, { PieChartContainerProps } from "./components/PieChartContainer";
+import { Alert } from "../components/Alert";
+
+declare function require(name: string): string;
+
+type VisibilityMap = {
+    [P in keyof PieChartContainerProps]: boolean;
+};
+
+// tslint:disable-next-line class-name
+export class preview extends Component<PieChartContainerProps, {}> {
+    render() {
+        return createElement("div", {},
+            createElement(Alert, {
+                className: `widget-${this.props.chartType}-chart-alert`,
+                message: PieChartContainer.validateProps(this.props)
+            }),
+            createElement(PieChart, {
+                config: {
+                    displayModeBar: this.props.showToolBar
+                },
+                data: [ {
+                    hole: this.props.chartType === "donut" ? .4 : 0,
```

Comment:
it's only the type that was specified on the feature sheet...maybe we can  ask the client 
https://docs.google.com/spreadsheets/d/1jfguDHL36jxeyZjxS8WWlc7DcCdwJO3Y5BWkuvSjbIM/edit#gid=0